backends/aoti/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Build AOTI backend for runtime.
+#
+# ### Editing this file ###
+#
+# This file should be formatted with
+# ~~~
+# cmake-format -i CMakeLists.txt
+# ~~~
+# It should also be cmake-lint clean.
+#
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+# Source root directory for executorch.
+if(NOT EXECUTORCH_ROOT)
+  set(EXECUTORCH_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../..)
+endif()
+
+# include(${EXECUTORCH_ROOT}/build/Utils.cmake)
+
+find_package(CUDAToolkit REQUIRED)
+
+set(_aoti_sources runtime/AotiBackend.cpp)
+add_library(aoti_backend STATIC ${_aoti_sources})
+target_include_directories(
+  aoti_backend
+  PUBLIC
+    ${CUDAToolkit_INCLUDE_DIRS}
+    $<BUILD_INTERFACE:${EXECUTORCH_ROOT}>
+    $<INSTALL_INTERFACE:include>
+)
+target_compile_options(aoti_backend PUBLIC -fexceptions -frtti -fPIC)
```

Comment:
-fexceptions and -frtti are default-on, no need to specify. -fPIC should be gated by CMAKE_POSITION_INDEPENDENT_CODE; why mess with it?